lib/raygun.breadcrumbs.ts

```diff
@@ -93,7 +93,7 @@ export function addBreadcrumb(
     lineNumber: callsite?.lineNumber || undefined,
   };
 
-  debug(`[raygun.breadcrumbs.ts] recorded breadcrumb: ${internalCrumb}`);
+  debug(`[raygun.breadcrumbs.ts] recorded breadcrumb: ${internalCrumb.message}`);
```

Comment:
Noticed that the original log would print as "recorded breadcrumb [object Object]" so instead I changed it to displaying the breadcrumb message.